Inquisitor.c

```diff
@@ -1,56 +1,60 @@
-//Inquisitor (Parsec Support tool)
+// Inquisitor (Parsec Support tool)
 
-#include <stdio.h> //File IO
-#include <stdlib.h> //Used for the Dynamic File Path
+#include <stdio.h> // File IO
+#include <stdlib.h> // Used for the Dynamic File Path
 #include <string.h> 
 
 
-//Gets & prints out the Parsec Log
-int getLog(){
-    //Open Log File for Reading
-    FILE* fp = fopen(strcat(getenv("USERPROFILE"),"/AppData/Roaming/Parsec/log.txt"),"r");
+/ /Gets & prints out the Parsec Log
+static void getLog(const char *path)
+{
+    // Open Log File for Reading
+    FILE* fp = fopen(path,"r");
     
-    //Check to make sure that the Parsec log oppened correctly
-    if(fp == NULL) {
-        printf("No Log Found \n");
+    // Check to make sure that the Parsec log opened correctly
+    if(fp == NULL)
+    {
+        printf("No Log Found in %s\n", path);
         return(-1);
     }
 
     //Section header
     printf("------------------------------PARSEC LOG------------------------------\n");
 
-    char c;
-    c = (char)fgetc(fp);
-    while (c != EOF){
+    // Print file contents
+    char c = fgetc(fp);
+    while (c != EOF)
+    {
         printf("%c", c);
         c = fgetc(fp);
     }
     
     printf("\n------------------------------END OF PARSEC LOG------------------------------\n\n");
 
-    int fclose(FILE* fp);//Close the log file
+    //Close the log file
+    fclose(FILE* fp);

```

Comment:
This made me realize that I did not correctly use fclose before. For this to work, the "File* " needs to be removed.